installer/win/Brackets_en-us.wxl

```diff
@@ -19,7 +19,7 @@
     <!-- See also: product name definitions in brackets-win-install-build.xml -->
     <String Id="ProductName">Brackets</String>
     <String Id="ShortProductName">Brackets</String>
-  
+    <String Id="smallProductName">brackets</String>
```

Comment:
Sorry for the late feedback, but all the other ids start with an uppercase letter while this one starts with a lowercase one.